src/components/header/header.js

```diff
@@ -1,18 +1,31 @@
-import React from "react";
+import React, { useState } from "react";
 import PropTypes from "prop-types";
 import "./header.scss";
 
+import Hamburger from "../../images/icons/hamburger.svg";
+import HamburgerOpen from "../../images/icons/hamburger-open.svg";
+
 export const Header = ({ name, Logo, MenuItems, ProfileItems }) => {
+  const [hamburgerOpened, hamburgerOnClick] = useState(true);
+  const HamburgerIcon = hamburgerOpened ? HamburgerOpen : Hamburger;
+
   return (
     <header className="App-header container">
       <div className="App-header__logo">
         {Logo}
-        {name ? <span className="App-header__title">{name}</span> : ""}
+        {name && <span className="App-header__title">{name}</span>}
       </div>
-      <nav>
+      {
+        <div className="App-header__hamburger">
```

Comment:
nit
```suggestion
        <div className="App-header__menu-icon">
```

Reply:
In general I would use terminology that describes the elements, not necessarily the type of the "element" in this case.